src/lamplib/src/genny/tasks/auto_tasks_local.py

```diff
@@ -0,0 +1,103 @@
+import os
+import shutil
+
+import structlog
+from git import Repo as GitRepo
+from shrub.command import CommandDefinition
+
+from genny.tasks.auto_tasks import (
+    ConfigWriter,
+    Repo,
+    WorkloadLister,
+    YamlReader,
+)
+from genny.tasks.auto_tasks_all import create_configuration, get_all_builds
+
+SLOG = structlog.get_logger("genny.tasks.auto_tasks.local")
+DSI_TMP_PATH = "./dsi"
+DSI_REPO_URL = "https://github.com/10gen/dsi.git"
+PRIVATE_WORKLOADS_TMP_PATH = "./PrivateWorkloads"
+PRIVATE_WORKLOADS_REPO_URL = "https://github.com/10gen/PrivateWorkloads.git"
+WORKLOAD_FILE_PATTERN = os.path.join("**", "src", "workloads", "**", "*.yml")
+FIRST_EXECUTION = 0
+WORKLOAD_PATH_KEY = "auto_workload_path"
+DSI_TASK_NAME = "f_run_dsi_workload"
+
+PROJECT_FILES = {
+    "master": [
+        os.path.join(DSI_TMP_PATH, "evergreen", "system_perf", "master", "variants.yml"),
```

Comment:
Won't these paths break when run in evergreen, since it will want dsi at `src/dsi`?